vimrc.bundles

```diff
@@ -16,7 +16,12 @@ call plug#begin('~/.vim/bundle')
 
 " Define bundles via Github repos
 Plug 'christoomey/vim-run-interactive'
-Plug 'ctrlpvim/ctrlp.vim'
+if executable('fzf')
+  Plug '/usr/local/opt/fzf'
```

Comment:
For me on Mac `fzf` is in `/usr/local/bin/fzf` (installed via HomeBrew).

Reply:
Ah, never mind. I thought this was supposed to be the path of the executable.

Reply:
@gylaz Hi! Is this working for you? It seems you've resolved your concern but I just want to check-in and make sure that's true. :)

Reply:
👋Thanks for checking in @SViccari! I got it working according with this PR.